extensions/vat-calculator/src/noviewnet.jsx

```diff
@@ -0,0 +1,20 @@
+import { Clipboard, showHUD, getPreferenceValues } from "@raycast/api";
+import { runAppleScript } from "@raycast/utils";
+
+import { getNetPrice } from "./lib/vat";
+
+export default async function quickVat(props) {
+  var number = getNetPrice(props.arguments.number);
+
+  if (getPreferenceValues().autopaste) {
+    showHUD(`Pasted Result to Active App`);
+    await runAppleScript(`
+      tell application "System Events"
+      keystroke "${number}" 
+      end tell
+      `);
```

Comment:
Why not use [Clipboard.paste](https://developers.raycast.com/api-reference/clipboard#clipboard.paste) here?